pex/bin/pex.py

```diff
@@ -721,8 +742,9 @@ def _safe_link(src, dst):
 def build_pex(reqs, options, cache=None):
     interpreters = None  # Default to the current interpreter.
 
-    pex_python_path = None  # Defaults to $PATH
-    if options.rc_file or not ENV.PEX_IGNORE_RCFILES:
+    pex_python_path = options.python_path  # If None, this will result in using $PATH.
+    # TODO(#1075): stop looking at PEX_PYTHON_PATH and solely consult the `--python-path` flag.
+    if not pex_python_path and (options.rc_file or not ENV.PEX_IGNORE_RCFILES):
```

Comment:
Should this be `if pex_python_path is not None`? What do we want the behavior to be if `--python-path` is explicitly set to an empty path?